File: fizzbuzz/src/javascript/fizzbuzz.js
for (let n = 1; n <= 50; n++) {
    if (n % 15 === 0) {
        process.stdout.write("fizz buzz\n");
    } else if (n % 3 == 0) {
        process.stdout.write("fizz\n");
    } else if (n % 5 === 0) {
        process.stdout.write("buzz\n");
    } else {
        process.stdout.write(`${n}\n`);
    }
}

File: fizzbuzz/src/swift/fizzbuzz.swift
for n in 1...50 {
    if n % 3 == 0 && n % 5 == 0 {
        print("fizz buzz")
    } else if n % 3 == 0 {
        print("fizz")
    } else if n % 5 == 0 {
        print("buzz")
    } else {
        print(n)
    }
}